README.md

```diff
@@ -98,6 +98,44 @@ payloads.
 />
 ```
 
+### Widget configuration props
+
+You can configure the state and behaviour of the widget with the following
+component props:
+
+- `uiMessageWebviewUrlScheme`: Used in postMessages and OAuth redirects.
+  Defaults to `mx`. See [OAuth redirects](#oauth-redirects) for additional
+  information.
+- `language`: Load the widget in the specified language. Defaults to `en-US`.
+  See [language
+  options](https://docs.mx.com/api#connect_configuring_connect_language_options)
+  for additional information.
+- `colorScheme`: Load the widget in the specified colorScheme; options are
+  `light` and `dark`. Defaults to `light`. _Connect only._
+- `currentInstitutionCode`: Load the widget into the credential view for the
+  specified institution. _Connect only._
+- `currentInstitutionGuid`: Load the widget into the credential view for the
+  specified institution. _Connect only._
+- `currentMemberGuid`: Load to a specific member that contains an error or
+  requires MFA from the most recent job. `currentMemberGuid` takes precedence
+  over `currentInstitutionCode`. _Connect only._
+- `disableInstitutionSearch`: When set to true, the institution search feature
+  will be disabled and end users will not be able to navigate to it. Must be
+  used with `currentInstitutionCode`, `currentInstituionGuid`, or
+  `currentMemberGuid`. _Connect only._
+- `includeTransactions`: When set to false while creating or updating a member,
+  transaction data will not be automatically aggregated. Future manual or
+  background aggregations will not be affected. Defaults to true. _Connect
+  only._
+- `updateCredentials`: Loads widget to the update credential view of a current
+  member. Optionally used with `currentMemberGuid`. This option should be used
+  sparingly. The best practice is to use `currentMemberGuid` and let the widget
+  resolve the issue. _Connect only._
+- `waitForFullAggregation`: Loads Connect, but forces the widget to wait until
+  any aggregation-type process is complete in order to fire a member connected
+  postMessage. This allows clients to have transactional data by the time the
+  widget is closed. _Connect only._
+
```

Comment:
Would it be worth it to have a connect specific section over the `_Connect only._` additions?

Reply:
@sambev I like that idea a lot. Here's what that looks like now:

<img width="855" alt="image" src="https://user-images.githubusercontent.com/601509/159802393-f5bef70d-8099-4efa-b9c6-d77793a4d8e3.png">